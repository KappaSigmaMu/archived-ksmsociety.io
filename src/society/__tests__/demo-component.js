import React from "react"
import renderer from "react-test-renderer"

import DemoComponent from "../demo-component"

describe("DemoComponent", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<DemoComponent />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
