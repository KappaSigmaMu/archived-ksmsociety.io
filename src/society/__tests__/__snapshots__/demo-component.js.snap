// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemoComponent renders correctly 1`] = `
<p>
  THIS IS A REACT COMPONENT
</p>
`;
